boot/modeenv.go

```diff
@@ -95,6 +99,8 @@ func ReadModeenv(rootdir string) (*Modeenv, error) {
 	m.Model = bm.model
 	// expect the caller to validate the grade
 	unmarshalModeenvValueFromCfg(cfg, "grade", &m.Grade)
+	unmarshalModeenvValueFromCfg(cfg, "current_boot_assets", &m.CurrentBootAssets)
```

Comment:
maybe we should call them "current_trusted_boot_assets"  and "current_recovery_trusted_boot_assets" to make it clear it's not all of them?

Reply:
Yes, that's a good point.